### PR TITLE
fix(ru): article <nav>

### PR DESCRIPTION
**Pull request by scffs**
As a rule, the MDN title looks like this <example element> - explanation

### CODE REVIEW

File: files/ru/web/html/element/nav/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Элемент секции навигации <nav>
+title: <nav> - Элемент навигации
```
Comment (Saionaro) [inline files/ru/web/html/element/nav/index.md:2]:
```suggestion
title: <nav> - элемент навигации
```

**Review by saionaro — APPROVED** [review]
@scffs спасибо за правку!

File: files/ru/web/html/element/nav/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Элемент секции навигации <nav>
+title: <nav> - элемент навигации
```
Comment (Saionaro) [inline files/ru/web/html/element/nav/index.md:2]:
```suggestion
title: <nav>: элемент навигации
```

File: files/ru/web/html/element/nav/index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Элемент секции навигации <nav>
+title: <nav>: элемент навигации
```
Comment (Saionaro) [inline files/ru/web/html/element/nav/index.md:2]:
```suggestion
title: `<nav>: элемент навигации`
```

File: files/ru/web/html/element/nav/index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Элемент секции навигации <nav>
+title: `<nav>: элемент навигации`
```
Comment (Saionaro) [inline files/ru/web/html/element/nav/index.md:2]:
```suggestion
title: <nav> - элемент навигации
```

**Review by saionaro — APPROVED** [review]

**Pull request merged**